languages/javascript/scripts/scaffold/scaffold.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "Scaffolding '$1'"
+
+cp -r ./template ../../exercises/concept/$1
```

Comment:
Would be _super_ helpfull if you take the first argument again and rename the files to that. in our case, concept name equals file name!